cloud_experts_tutorials/cloud-experts-dynamic-certificate-custom-domain.adoc

```diff
@@ -19,16 +19,16 @@ toc::[]
 
 While wildcard certificates provide simplicity by securing all first-level subdomains of a given domain with a single certificate, other use cases can require the use of individual certificates per domain.
 
-Learn how to use the link:https://docs.openshift.com/container-platform/latest/security/cert_manager_operator/index.html[cert-manager Operator for Red{nbsp}Hat OpenShift] and link:https://letsencrypt.org/[Let's Encrypt] to dynamically issue certificates for routes created using a custom domain.
+Learn how to use the link:https://docs.openshift.com/container-platform/latest/security/cert_manager_operator/index.html[cert-manager Operator for Red Hat OpenShift] and link:https://letsencrypt.org/[Let's Encrypt] to dynamically issue certificates for routes created using a custom domain.
```

Comment:
Can you keep the `{nbsp}` between Red and Hat? It keeps the two words together, and we made this change recently.

Reply:
I honestly thought this was a mistake so that's why I removed it! I added it back!